### PR TITLE
Search Input | Hotfix for overriding paddings coming from auth styles

### PR DESCRIPTION
**Pull request by bkoval**
@rogatty @vforge It looks like it's working for me but if we agree on proceeding with this hotfix, let's create a ticket for QA-ing this.

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: front/styles/main/component/_side-nav.scss [hunk 1]
```diff
@@ -65,8 +66,7 @@
 			left: 0;
 			margin: rem-calc(8) rem-calc(13);
 			outline: none;
-			padding-left: rem-calc(95);
-			padding-right: rem-calc(25);
+			padding: 0.5rem rem-calc(25) 0.5rem rem-calc(95);
```
Comment (vforge) [inline front/styles/main/component/_side-nav.scss]: please use following:
```
padding: {
  top: X;
  bottom: X;
```

**Comment by vforge** [comment]
minor issue, lgtm if fixed

**Comment by vforge** [comment]
LGTM :+1:

**Comment by bkoval** [comment]
@rogatty @justnpT was this tested by any chance?

**Comment by rogatty** [comment]
@bkoval it wasn't, please create a ticket in XW project.

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**